### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Lavanyagaur22**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,16 +10,7 @@ fully featured Offline Workflow and Conflict Resolution.
...
-
-#### Steps Followed:
-A database is maintained which stores the class name of the mutation and the value map of variables given by user along with some other details like operationID, etc.
-
-- When the client comes online, clicking on the "FROMDB" button will take mutation from DB by iterating over a list of mutations stored in the offline queue.
-- We extract the class name of the mutation and the value map variables.
-- Then we take the constructor of the class and the parameters that the constructor takes in.
-- Then we have stored the values of json in an array list of objects and pass this arraylist as an argument in the constructor.
-- Then we make an object of mutation and make a call by passing the above object in mutation call.
+## Offline Support is implemented where the mutations are persisted in an offline queue when the client is offline.
```
Comment (wtrocki) [inline README.md]: Kinda long header :) In headers use capitals

File: README.md [hunk 2]
```diff
@@ -37,7 +28,7 @@ See `sample` for example application. For now it's using ionic showcase as its b
 - Now clone this repository (https://github.com/aerogear/voyager-android.git)
 - Run the application to send query and mutation to the server and displaying to the user.
 
-### To check offline part:
+### Try offline capabilities of Offix-Android:
```
Comment (wtrocki) [inline README.md]: no : in headers

File: README.md [hunk 3]
```diff
@@ -37,8 +28,13 @@ See `sample` for example application. For now it's using ionic showcase as its b
...
 
-### To check offline part:
+## Try offline capabilities of Offix-Android
 
 1. Make a mutation by going offline.
 2. Then, come online and click on "FromDb" button and you will see the results.
 3. Also, refresh the application to see the updated cache.
+
+
+#### Here is a short demo of the offline capabilites where the client performs a mutation of updating the Task3 and when it comes online again, changes are refelected back in the application.
```
Comment (harshithdwivedi) [inline README.md]: Header's too long, please shorten it.
Have a look at the issue I opened #41 on how to write headers

**Review by harshithdwivedi — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -37,8 +28,16 @@ See `sample` for example application. For now it's using ionic showcase as its b
...
 
-### To check offline part:
+## Try offline capabilities of Offix-Android
 
 1. Make a mutation by going offline.
 2. Then, come online and click on "FromDb" button and you will see the results.
 3. Also, refresh the application to see the updated cache.
+
+
+### Short demo of the offline capabilites
```
Comment (harshithdwivedi) [inline README.md]: Just "Demo" will do

File: README.md [hunk 5]
```diff
@@ -37,8 +28,16 @@ See `sample` for example application. For now it's using ionic showcase as its b
 - Now clone this repository (https://github.com/aerogear/voyager-android.git)
 - Run the application to send query and mutation to the server and displaying to the user.
 
-### To check offline part:
+## Try offline capabilities of Offix-Android
```
Comment (harshithdwivedi) [inline README.md]: "Running the sample app"

File: README.md [hunk 6]
```diff
@@ -10,16 +10,7 @@ fully featured Offline Workflow and Conflict Resolution.
...
-
-#### Steps Followed:
-A database is maintained which stores the class name of the mutation and the value map of variables given by user along with some other details like operationID, etc.
-
-- When the client comes online, clicking on the "FROMDB" button will take mutation from DB by iterating over a list of mutations stored in the offline queue.
-- We extract the class name of the mutation and the value map variables.
-- Then we take the constructor of the class and the parameters that the constructor takes in.
-- Then we have stored the values of json in an array list of objects and pass this arraylist as an argument in the constructor.
-- Then we make an object of mutation and make a call by passing the above object in mutation call.
+#### Offline Support is implemented in the application where the mutations are persisted when the client goes offline.
```
Comment (harshithdwivedi) [inline README.md]: "Imppementing offline support"

**Review by harshithdwivedi — CHANGES_REQUESTED** [review]
Please look at the comments

File: README.md [hunk 7]
```diff
@@ -10,16 +10,7 @@ fully featured Offline Workflow and Conflict Resolution.
...
-
-#### Steps Followed:
-A database is maintained which stores the class name of the mutation and the value map of variables given by user along with some other details like operationID, etc.
-
-- When the client comes online, clicking on the "FROMDB" button will take mutation from DB by iterating over a list of mutations stored in the offline queue.
-- We extract the class name of the mutation and the value map variables.
-- Then we take the constructor of the class and the parameters that the constructor takes in.
-- Then we have stored the values of json in an array list of objects and pass this arraylist as an argument in the constructor.
-- Then we make an object of mutation and make a call by passing the above object in mutation call.
+#### Implemented offline support.
```
Comment (harshithdwivedi) [inline README.md]: Remove this, it's not needed for now.
Also get rid of the features that are not implemented yet for example conflict resolution, subscription, etc.

**Comment by Lavanyagaur22** [comment]
@harshithdwivedi Please have a look.

**Review by harshithdwivedi — APPROVED** [review]

**Pull request merged**